compression_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connect
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/bufbuild/connect-go/internal/assert"
+	"google.golang.org/protobuf/types/known/emptypb"
```

Comment:
Unrelated: TIL the emptypb, this is useful with something else I was testing.

Reply:
It's the weirdest (and therefore my favorite) of the well-known types :)

I end up using a lot in cases like this.